### PR TITLE
Replace occurrances of DAML SDK in the docs with appropriate new terms

### PR DESCRIPTION
**Pull request by bame-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/configs/pdf/index.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Getting started
    :titlesonly:
    :maxdepth: 2
 
-   Installing the SDK <getting-started/installation>
+   Installing DAML Connect <getting-started/installation>
```
Comment (cocreature) [inline docs/configs/pdf/index.rst:14]: Given that we renamed the thing installed by the assistant to "DAML SDK", I think this should also be DAML SDK.

**Review by cocreature — CHANGES_REQUESTED** [review]
Mostly looks good to me but this needs some page that explains the relationship between DAML connect and SDK. Otherwise this is just very confusing to users

File: docs/configs/pdf/index.rst (hunk 1 above)
Comment (bame-da) [inline docs/configs/pdf/index.rst:14]: Fair enough.

```suggestion
   Installing the SDK <getting-started/installation>
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ DAML SDK documentation
    :hidden:
    :caption: Getting started
 
-   Installing the SDK <getting-started/installation>
+   Installing DAML Connect <getting-started/installation>
```
Comment (bame-da) [inline docs/source/index.rst:20]:
```suggestion
   Installing the SDK <getting-started/installation>
```

**Comment by bame-da** [comment]
> Mostly looks good to me but this needs some page that explains the relationship between DAML connect and SDK. Otherwise this is just very confusing to users

Fully agree. A second PR with an explanation of what DAML Connect is will follow. Just doing this piecemeal to keep PRs managable. This is the pure renaming PR.

**Comment by cocreature** [comment]
> Fully agree. A second PR with an explanation of what DAML Connect is will follow. Just doing this piecemeal to keep PRs managable. This is the pure renaming PR.

Fair enough, I’m generally not super fond of splitting up PRs if it introduces inconsistencies (master should always be releasable) but I’ll trust you that you can get the change in before 1.7 :slightly_smiling_face:

**Review by cocreature — APPROVED** [review]
Thanks!

**Pull request merged**